org-code-javabuilder/playground/src/main/java/org/code/playground/Playground.java

```diff
@@ -0,0 +1,23 @@
+package org.code.playground;
+
+import java.util.HashMap;
+import org.code.protocol.*;
+
+public class Playground {
+  private final OutputAdapter outputAdapter;
+
+  public Playground() {
+    this.outputAdapter = GlobalProtocol.getInstance().getOutputAdapter();
+  }
+
+  public void sendTestMessages() {
+    HashMap<String, String> details = new HashMap<>();
+    details.put("filename", "puppy_1.jpg");
+    details.put("width", "200");
+    details.put("height", "200");
+    details.put("x", "200");
+    details.put("y", "200");
+    details.put("id", "first_puppy");
+    this.outputAdapter.sendMessage(new PlaygroundMessage(PlaygroundSignalKey.ADD_IMAGE, details));
+  }
```

Comment:
nit: we can remove this test code (or remove it when you implement the no-op API)

Reply:
Yeah I'll remove then, was imagining this PR as just a way to get something that is runnable, and will clean up with no-op API.